docs/translating.md

```diff
@@ -84,12 +85,85 @@ The second step is to upload the translatable strings to Transifex using
 The final step is to get the translated files from Transifex using
 `tx pull -a`.
 
```

Comment:
What's the end result? What are some example filenames & file extensions I'll see?